<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilloutDlg</class>
 <widget class="QDialog" name="FilloutDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>281</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>281</width>
    <height>321</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblChart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tekst in de flowchart:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code-tekst:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leChart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Algemene code invoegen</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFunc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Een functie van een apparaat aanroepen</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInst">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Van decaratie</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbInst">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFunc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFunc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Functie</string>
   </property>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leChart</tabstop>
  <tabstop>rbFunc</tabstop>
  <tabstop>cbInst</tabstop>
  <tabstop>cbFunc</tabstop>
  <tabstop>rbCode</tabstop>
  <tabstop>pbOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
